content/pt-br/docs/concepts/configuration/secret.md

```diff
@@ -0,0 +1,2141 @@
+---
+title: Secrets
+content_type: concept
+feature:
+  title: Secrets e gerenciamento de configuração
+  description: >
+    Crie e atualize Secrets e configurações da aplicação sem reconstruir sua imagem
+    de contêiner e sem expor credenciais na configuração da sua aplicação.
+weight: 30
+---
+
+<!--
```

Comment:
I think we can remove these comments with the original content :)

Reply:
Will do!

Reply:
Removed!